test/e2e/sagemaker/replacement_values.py

```diff
@@ -67,9 +67,62 @@
+    "us-west-2": "159807026194.dkr.ecr.us-west-2.amazonaws.com",
+    "af-south-1": "875698925577.dkr.ecr.af-south-1.amazonaws.com",
+    "ap-east-1": "001633400207.dkr.ecr.ap-east-1.amazonaws.com",
+    "ap-northeast-1": "574779866223.dkr.ecr.ap-northeast-1.amazonaws.com",
+    "ap-northeast-2": "709848358524.dkr.ecr.ap-northeast-2.amazonaws.com",
+    "ap-south-1": "126357580389.dkr.ecr.ap-south-1.amazonaws.com",
+    "ap-southeast-1": "245545462676.dkr.ecr.ap-southeast-1.amazonaws.com",
+    "ap-southeast-2": "563025443158.dkr.ecr.ap-southeast-2.amazonaws.com",
+    "ca-central-1": "536280801234.dkr.ecr.ca-central-1.amazonaws.com",
+    "cn-north-1": "453000072557.dkr.ecr.cn-north-1.amazonaws.com.cn",
+    "cn-northwest-1": "453252182341.dkr.ecr.cn-northwest-1.amazonaws.com.cn",
+    "eu-central-1": "048819808253.dkr.ecr.eu-central-1.amazonaws.com",
+    "eu-north-1": "895015795356.dkr.ecr.eu-north-1.amazonaws.com",
+    "eu-south-1": "933208885752.dkr.ecr.eu-south-1.amazonaws.com",
+    "eu-west-1": "468650794304.dkr.ecr.eu-west-1.amazonaws.com",
+    "eu-west-2": "749857270468.dkr.ecr.eu-west-2.amazonaws.com",
+    "eu-west-3": "680080141114.dkr.ecr.eu-west-3.amazonaws.com",
+    "me-south-1": "607024016150.dkr.ecr.me-south-1.amazonaws.com",
+    "sa-east-1": "539772159869.dkr.ecr.sa-east-1.amazonaws.com",
+    "us-gov-west-1": "362178532790.dkr.ecr.us-gov-west-1.amazonaws.com",
+}
+
+# https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-configure-processing-jobs.html#clarify-processing-job-configure-container
+CLARIFY_IMAGE_URIS = {
+    "us-east-1": "205585389593.dkr.ecr.us-east-1.amazonaws.com",
+    "us-east-2": "211330385671.dkr.ecr.us-east-2.amazonaws.com",
+    "us-west-1": "740489534195.dkr.ecr.us-west-1.amazonaws.com",
+    "us-west-2": "306415355426.dkr.ecr.us-west-2.amazonaws.com",
+    "ap-east-1": "098760798382.dkr.ecr.ap-east-1.amazonaws.com",
+    "ap-south-1": "452307495513.dkr.ecr.ap-south-1.amazonaws.com",
+    "ap-northeast-2": "263625296855.dkr.ecr.ap-northeast-2.amazonaws.com",
+    "ap-southeast-1": "834264404009.dkr.ecr.ap-southeast-1.amazonaws.com",
+    "ap-southeast-2": "007051062584.dkr.ecr.ap-southeast-2.amazonaws.com",
+    "ap-northeast-1": "377024640650.dkr.ecr.ap-northeast-1.amazonaws.com",
+    "ca-central-1": "675030665977.dkr.ecr.ca-central-1.amazonaws.com",
+    "eu-central-1": "017069133835.dkr.ecr.eu-central-1.amazonaws.com",
+    "eu-west-1": "131013547314.dkr.ecr.eu-west-1.amazonaws.com",
+    "eu-west-2": "440796970383.dkr.ecr.eu-west-2.amazonaws.com",
+    "eu-west-3": "341593696636.dkr.ecr.eu-west-3.amazonaws.com",
+    "eu-north-1": "763603941244.dkr.ecr.eu-north-1.amazonaws.com",
+    "me-south-1": "835444307964.dkr.ecr.me-south-1.amazonaws.com",
+    "sa-east-1": "520018980103.dkr.ecr.sa-east-1.amazonaws.com",
+    "af-south-1": "811711786498.dkr.ecr.af-south-1.amazonaws.com",
+    "eu-south-1": "638885417683.dkr.ecr.eu-south-1.amazonaws.com"
+}
```

Comment:
I know you're just copying the previous code style, but I'd like to point out that none of the above are "image URIs" :) They are ECR endpoint URIs.

Reply:
... hmm